src/collection-browser.ts

```diff
@@ -263,9 +263,16 @@ export class CollectionBrowser
      *
      * We disable it during the automated scroll since we don't want to fetch pages for intervening cells the
      * user may never see.
+     *
+     * No need to fetch if placeholder is being displayed
      */
-    if (!model && !this.isScrollingToCell) {
+    if (!this.placeholderType && !model && !this.isScrollingToCell) {
```

Comment:
this block fires even though we render the placeholder. this reduces fetch checks since we request details for each result...